pkgs/development/tools/database/pgweb/default.nix

```diff
@@ -0,0 +1,15 @@
+{ buildGoPackage, fetchFromGitHub, ... }:
+
+buildGoPackage rec {
+  pname = "pgweb";
+  version = "0.11.7";
+
+  goPackagePath = "github.com/sosedoff/pgweb";
+
+  src = fetchFromGitHub {
+    owner = "sosedoff";
+    repo = "pgweb";
+    rev = "v${version}";
+    sha256 = "1df3vixxca80i040apbim80nqni94q882ykn3cglyccyl0iz59ix";
+  };
```

Comment:
Having meta is not nice to have but required.

Reply:
👍